routes/index.js

```diff
@@ -0,0 +1,5 @@
+const express = require("express");
```

Comment:
Do you use the router module somewhere? I don't see it being imported into the server index.js anywhere.

Reply:
No, I'm not using it. Forgot to remove it. Thanks for the comment!